streams/us.m3u

```diff
@@ -760,6 +760,10 @@ https://www.livedoomovie.com/02_FoxThai_TH_720p/chunklist.m3u8
 https://mn-nl.mncdn.com/blutv_foxtv2/live.m3u8 
 #EXTINF:-1 tvg-id="FoxWeather.us" status="online",Fox Weather (720p)
 https://247wlive.foxweather.com/stream/index.m3u8
+#EXTINF:-1 tvg-id="FoxWeather.us",Fox Weather
+https://foxweather-xumo.amagi.tv/playlist.m3u8
+#EXTINF:-1 tvg-id="FoxWeather.us",Fox Weather
+https://dai2.xumo.com/amagi_hls_data_xumo1212A-foxweather-xumo/CDN/master.m3u8
```

Comment:
We have a separate file for streams from the xumo service: https://github.com/iptv-org/iptv/blob/master/streams/us_xumo.m3u. Please move these links to it.

Reply:
Will do this afternoon.